podpac/core/coordinates/array_coordinates1d.py

```diff
@@ -281,6 +281,9 @@ def _get_definition(self, full=True):
     # ------------------------------------------------------------------------------------------------------------------
 
     def _select(self, bounds, return_indices, outer):
```

Comment:
@mpu-creare Why is it that we use `lower_precision_time_bounds` for `UniformCoordinates1d._select` but `higher_precision_coordinates` for `ArrayCoordinates1d.select`? Seems like it should be the same, and we should be able to move that up into the `Coordinates1d.select` method.

https://github.com/creare-com/podpac/blob/514dc5da3975fa1c6051f0da1691cf5ba4972bf8/podpac/core/coordinates/uniform_coordinates1d.py#L434

Reply:
> Seems like it should be the same, and we should be able to move that up into the top of the Coordinates1d.select method.

Oh, we do use `lower_precision_time_bounds` in `Coordinates1d.select`. It is probably redundant in `UniformCoordinates1d._select`. The ArrayCoordinates1d has to use the higher precision coordinates  for comparison to the actual coordinates. I get it now.